app/javascript/components/explore/StudyGeneField.jsx

```diff
@@ -160,7 +160,13 @@ export default function StudyGeneField({ genes, searchGenes, allGenes, speciesLi
       <div className="flexbox align-center">
         <div className="input-group">
           <div className="input-group-append">
-            <Button type="button" data-analytics-name="gene-search-submit" onClick={handleSearch} disabled={searchDisabled}>
+            <Button
+              type="button"
+              aria-label="Search genes"
+              aria-disabled={searchDisabled}
```

Comment:
@ehanna4 I'm excited to see this a11y PR-- thank you for chipping away at some of these things! A couple of questions--

1) I'm not seeing that this is deployed yet. When will that happen?
2) If one is using the button html tag, it isn't necessary to add `aria-disabled` because screen readers will pick up on the disabled property of the native element. So are these not actually rendered as buttons?

Reply:
Release information can be found in the Releases tab on this repo: https://github.com/broadinstitute/single_cell_portal_core/releases all PRs are (supposed to be) added to the appropriate release. 

Single Cell Portal generally releases this repo every 2 weeks, on a Wednesday, with our next one scheduled for 5/3/2023

I'm curious what drew your attention to this PR and your interest in it's release?

Reply:
Thanks for the information! I'm just poking around SCP today to learn a bit about the UI. I am curious about the level of accessibility support/requirements for all of our non-Terra UI code, and this PR was exciting to me because it indicates that y'all are thinking about a11y.

Reply:
Gotcha, we have some [tickets](https://broadworkbench.atlassian.net/browse/SCP-4024?atlOrigin=eyJpIjoiNzcyMmYxMDlhZGNlNDA4ZWI3YjI5ZTc3NzhlNDE0YTEiLCJwIjoiaiJ9) in our backlog that seem to suggest Gabe de Mattos has done some amount of looking at SCPs accessibility (unsure how deep of a dive). I'm not sure how much you are digging in but he might have more insight as well.